package.json

```diff
@@ -1,65 +1,71 @@
 {
     "name": "web-mapviewer-monorepo",
     "private": true,
-    "workspaces": [
-        "packages/geoadmin-log",
-        "packages/geoadmin-numbers",
-        "packages/geoadmin-coordinates",
-        "packages/mapviewer"
-    ],
     "scripts": {
-        "dev": "npm run dev --workspace=web-mapviewer",
-        "build": "npm run build --workspaces",
-        "build:dev": "npm run build:dev --workspaces",
-        "build:int": "npm run build:int --workspaces",
-        "build:prod": "npm run build:prod --workspaces",
-        "preview": "npm run build --workspaces && npm run preview --workspace=web-mapviewer",
-        "preview:dev": "npm run build:dev --workspaces && npm run preview:dev --workspace=web-mapviewer",
-        "preview:int": "npm run build:int --workspaces && npm run preview:int --workspace=web-mapviewer",
-        "preview:prod": "npm run build:prod --workspaces && npm run preview:prod --workspace=web-mapviewer",
-        "test:unit": "npm run test:unit --workspaces --if-present",
-        "test:unit:watch": "npm run test:unit:watch --workspaces --if-present",
-        "test:e2e": "npm run test:e2e --workspace=web-mapviewer",
-        "test:e2e:ci": "npm run test:e2e:ci --workspace=web-mapviewer",
-        "test:e2e:headless": "npm run test:e2e:headless --workspace=web-mapviewer",
-        "test:component": "npm run test:component --workspace=web-mapviewer",
-        "test:component:ci": "npm run test:component:ci --workspace=web-mapviewer",
+        "dev": "pnpm run dev --workspace=web-mapviewer",
+        "build-all": "pnpm run -r build",
+        "build:dev": "pnpm -r run build:dev",
+        "build:int": "pnpm -r run build:int",
+        "build:prod": "pnpm -r run build:prod",
+        "preview": "pnpm run -r --filter=!web-mapviewer build --filter= && pnpm run -r --filter=web-mapviewer preview",
```

Comment:
there's an empty `--filter=` here, is that wanted?

Reply:
Nope that's a mistake